src/components/studies/scheduler/SchedulableSingleSessionContainer.tsx

```diff
@@ -216,6 +216,14 @@ const SchedulableSingleSessionContainer: FunctionComponent<SchedulableSingleSess
     return index + result + messageKey
   }
 
+  const canDeleteNotification = (notifications: ScheduleNotification[], index: number) => {
+    // follow-up notifications are always deleteable
+    if (index > 0) return true
```

Comment:
I think it's easier to read  as `return index> 0 || notifications.length ===1`

Reply:
Agreed! Made the change in the next commit